www/data/docs_sidebar.yml

```diff
@@ -1,122 +1,195 @@
 ---
 sidebar_links:
-- title: Getting Started
+- title: 
   links:
-  - title: Overview
-    link: "/docs/overview/"
-  - title: Download and install Habitat
-    link: "/tutorials/download/"
-  - title: Tutorials
-    link: "/tutorials/"
-  - title: Concepts <span>&plus;</span>
-    link: "/docs/concepts-overview/"
+  - title: Docs Home
+    link: "/docs"
+  - title: Using Habitat <span>&plus;</span>
+    link: "/docs/using-habitat"
     sub_links:
+    - title: Overview
+      link: "#overview"
+    - title: Installing Habitat
+      link: "#install-habitat"
+    - title: Install FAQ
+      link: "/docs/install-faq"
+    - title: Using Habitat
+      link: "#using-habitat"
+    - title: Service Groups
+      link: "#service-groups"
+    - title: Topologies
+      link: "#topologies"
+    - title: Config Updates
+      link: "#config-updates"
+    - title: File Uploads
+      link: "#file-uploads"
+    - title: Encryption
+      link: "#using-encryption"
+    - title: Update Strategies
+      link: "#using-updates"
+  - title: Developing Packages <span>&plus;</span>
+    link: "/docs/developing-packages"
+    sub_links:
+    - title: Writing a Plan.sh
+      link: "#write-plans"
+    - title: Config Templates
+      link: "#add-configuration"
+    - title: Binds and Exports
+      link: "#pkg-binds"
+    - title: Plan Builds
+      link: "#plan-builds"
+    - title: Troubleshooting Builds
+      link: "#debug-builds"
+    - title: Package Post-Processors
```

Comment:
this is the only place we call this a post-processor, so it feels inconsistent. How about "Export package formats" or something

Reply:
Corrected by new commit.